modules/deploy/pages/deployment-option/cloud/create-byoc-cluster-azure.adoc

```diff
@@ -0,0 +1,54 @@
+= Create a BYOC Cluster on Azure
+:description: Use the Redpanda Cloud UI to create a BYOC cluster on Azure.
+:page-cloud: true
+
+NOTE: BYOC on Azure is currently in a limited availability (LA) release. Additional functionality is expected when it is generally available.
+
+== Prerequisites
+
+* A minimum version of `rpk` v24.1
+* A dedicated Azure subscription with the following resources: 
++
+** Resources:
++
+*** Microsoft.Compute
+*** Microsoft.ManagedIdentity
+*** Microsoft.Storage
+*** Microsoft.KeyVault
+*** Microsoft.Network
+*** Microsoft.Compute.EncryptionAtHost
++
+** Quota: A tier 1 Redpanda cluster requires the following:
++
+*** Standard Lasv3-series vCPUs: 24
+*** Standard Dadsv5-series vCPUs: 8
++
+** Role: The Azure user must have the *Owner* role.
+
+== Create a BYOC cluster
+
+To create a Redpanda cluster in your virtual private cloud (VPC), log in to https://cloud.redpanda.com[Redpanda Cloud^]. 
+
+. On the Clusters page, click *Create cluster*, then click *Create BYOC cluster*.
+. Enter a cluster name, then select the resource group, provider (Azure), xref:deploy:deployment-option/cloud/byoc-tiers.adoc[region, availability zones, tier], and Redpanda version. 
++
+[NOTE]
+==== 
+* If you plan to create a private network in your own VPC, select the region where your VPC is located.
+* Three availability zones provide two backups in case one availability zone goes down.
+====
++ 
+Optionally, click *Advanced settings* to specify up to five key-value custom tags. After the cluster is created, the tags are applied to all Azure resources associated with this cluster. For more information, see the https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources[Azure documentation^].
+
+. Click *Next*.
+. On the Network page, enter the connection type: either *Public* or *Private*. For BYOC clusters, *Private* is best-practice.
```

Comment:
I am not sure if you want to mention it but we won't have support for peering out of the box so you might want to mention that the only supported path for the time being is private link. Just a thought